src/ar/unlam/edu/ar/tp/model/cazador/CazadorUrbano.java

```diff
@@ -0,0 +1,28 @@
+package ar.unlam.edu.ar.tp.model.cazador;
+
+import ar.unlam.edu.ar.tp.model.profugo.Profugo;
+
+public class CazadorUrbano extends CazadorBase {
+	
+	public CazadorUrbano(String nombre, int experiencia) {
+		super(nombre, experiencia);
+	}
+	
+	@Override
+	public boolean puedeCapturar(Profugo p) {
+		boolean sePuedeCapturar = false; 
+		
+		if( this.getExperiencia() > p.getInocencia() && 
+			p.isNervioso() == false ) {
+			sePuedeCapturar = true;
+		}
+		
+		return sePuedeCapturar;
```

Comment:
```suggestion
		return this.getExperiencia() > p.getInocencia() && !p.isNervioso();
```
Se puede simplificar